src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-gcloud.py

```diff
@@ -0,0 +1,14 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+from PyInstaller.utils.hooks import copy_metadata
+datas = copy_metadata('gcloud')
```

Comment:
Can we have a trailing newline to keep the linter happy.

Reply:
Tks for the advice. let me make a change.